frontend/src/tournaments/LeaderboardTab.tsx

```diff
@@ -114,30 +136,85 @@ const LeaderboardTab = () => {
                         select
                         label='Time Control'
                         value={timeControl}
+                        color='primary'
                         onChange={(e) => setTimeControl(e.target.value as TimeControl)}
                     >
-                        <MenuItem value={'blitz'}>Blitz</MenuItem>
-                        <MenuItem value={'rapid'}>Rapid</MenuItem>
-                        <MenuItem value={'classical'}>Classical</MenuItem>
+                        <MenuItem value={'blitz'}>
+                            <Icon
+                                name={TimeControlType.Blitz}
+                                sx={{ verticalAlign: 'middle', marginRight: 1 }}
+                                color={getColor(TimeControlType.Blitz)}
+                            />{' '}
+                            Blitz
+                        </MenuItem>
+                        <MenuItem value={'rapid'}>
+                            <Icon
+                                name={TimeControlType.Rapid}
+                                sx={{ verticalAlign: 'middle', marginRight: 1 }}
+                                color={getColor(TimeControlType.Rapid)}
+                            />
+                            Rapid
+                        </MenuItem>
+                        <MenuItem value={'classical'}>
+                            <Icon
+                                name={TimeControlType.Classical}
+                                sx={{ verticalAlign: 'middle', marginRight: 1 }}
+                                color={getColor(TimeControlType.Classical)}
+                            />
+                            Classical
+                        </MenuItem>
                     </TextField>
 
                     <TextField
                         data-cy='tournament-type-selector'
                         sx={{ minWidth: 130 }}
                         select
                         label='Tournament Type'
+                        color='primary'
                         value={tournamentType}
                         onChange={(e) =>
                             setTournamentType(e.target.value as TournamentType)
                         }
                     >
-                        <MenuItem value={TournamentType.Arena}>Arena</MenuItem>
-                        <MenuItem value={TournamentType.Swiss}>Swiss</MenuItem>
-                        <MenuItem value={TournamentType.GrandPrix}>Grand Prix</MenuItem>
+                        <MenuItem value={TournamentType.Arena}>
+                            {' '}
+                            <Icon
+                                name={'Arena'}
```

Comment:
You don't have to change all of them, but just for future reference, if you're just passing a string, you don't need the braces. So you can just do `name='Arena'` instead of `name={'Arena'}`. The braces are only necessary if you're passing a TypeScript value/expression

Reply:
sure